components/typography/stories/template.js

```diff
@@ -106,7 +112,413 @@ export const Template = (args = {}, context = {}) => {
 	/** Wrap items with the spectrum-Typography wrapper if there are more than 1 items (this ensures correct margins) */
 	return html`${when(
 		content.length > 1,
-		() => html`<div class="spectrum-Typography" id=${ifDefined(id)}>${processedContent}</div>`,
+		() => html`<div class="spectrum-Typography" id=${ifDefined(id)} lang=${lang || nothing}>${processedContent}</div>`,
```

Comment:
The ifDefined syntax would be the best practice here or you could use the fallback of the global lang value:

```suggestion
		() => html`<div class="spectrum-Typography" id=${ifDefined(id)} lang=${ifDefined(lang)}>${processedContent}</div>`,
```